src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java

```diff
@@ -231,7 +231,7 @@ public boolean equals(Object o) {
     if (this == o) return true;
     if (o == null || getClass() != o.getClass()) return false;
     FileColumnProperties that = (FileColumnProperties) o;
-    return Objects.equals(columnOrdinal,that.columnOrdinal)
+    return Objects.equals(columnOrdinal, that.columnOrdinal)
```

Comment:
My recent format.sh script suggested me these changes as well. 
ahh do you know why it passed in previous PR?

Reply:
not really, i don't know why it passed the format gate